week1/김상진/sangjin-namespace.yaml

```
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata
```

yaml 파일 형식에 안 맞습니다. 아래와 같이 수정해 주세요.
(suggestion commit 하기 버튼 누루시면 바로 아래와 같이 반영됩니다.)
```suggestion
metadata:
```